### ISSUE TITLE
robyn_refresh "Error: ValueError: zero-size array to reduction operation minimum which has no identity"

### ISSUE THREAD

**Issue by ChrisHaferl** [facebookexperimental/Robyn#761]
## Describe issue
On robyn_refresh I am getting an error of "Error: ValueError: zero-size array to reduction operation minimum which has no identity"

## Provide reproducible example
```{r}
library(Robyn) #Using Robyn_3.10.3
```

```{r}
#Load data used to build model + new data
dt_input_data <- read.csv(file = '~/Documents/data-scratch/chrishaferl/MMMv2/USA_Model/Robyn_202306140111_init/raw_data.csv')
```

```{r}
#Load saved model, located in model output folder
json_file <- "~/Documents/data-scratch/chrishaferl/MMMv2/USA_Model/Robyn_202306140111_init/RobynModel-6_95_13.json"
```

```{r}
RobynRefresh <- robyn_refresh(
  json_file = json_file,
  dt_input = dt_input_data,
  dt_holidays = dt_prophet_holidays,
  refresh_steps = 12, #Number of weeks to add
  refresh_iters = 1000, # 1k is an estimation
  refresh_trials = 1
)
```

Output:
```
>>> Recreating model 6_95_13
Imported JSON file succesfully: ~/Documents/data-scratch/chrishaferl/MMMv2/USA_Model/Robyn_202306140111_init/RobynModel-6_95_13.json
>> Running feature engineering...
Input data has 178 weeks in total: 2020-01-06 to 2023-05-29
Initial model is built on rolling window of 84 week: 2021-05-31 to 2023-01-02
>>> Calculating response curves for all models' media variables (9)...
Successfully recreated model ID: 6_95_13

>>> Building refresh model #1 in manual mode
>>> New bounds freedom: 14.29%
>> Running feature engineering...
Input data has 178 weeks in total: 2020-01-06 to 2023-05-29
Refresh #1 model is built on rolling window of 84 week: 2021-08-23 to 2023-03-27
Rolling window moving forward: 12 weeks
Warning: Provided train_size but ts_validation = FALSE. Time series validation inactive.Fitting time series with all available data...
Using weibull_pdf adstocking with 46 hyperparameters (0 to iterate + 46 fixed) on 15 cores
>>> Starting 1 trials with 1000 iterations each using TwoPointsDE nevergrad algorithm...
  Running trial 1 of 1
Error: ValueError: zero-size array to reduction operation minimum which has no identity

```
## Environment & Robyn version
Using Robyn_3.10.3

**Comment by gufengzhou** [comment]
Hi, I can't reproduce your error. I tested it with both geometric and weibull using the simulated data through the  1st refresh and everything works. I noticed that

**Comment by ChrisHaferl** [comment]
@gufengzhou Right, that is odd, attached the model JSON (had to save as txt as json upload not supported here)

[RobynModel-6_95_13.txt](https://github.com/facebookexperimental/Robyn/files/12147732/RobynModel-6_95_13.txt)

**Comment by rsplima** [comment]
Any news on this? I also get this exact error message when running robyn_refresh (I've tried editing my JSON file and setting `hyper_fixed = 'false'`, without success).

**Comment by tiagoyuhang** [comment]
I am getting the same problem. One of the things I have noticed is that the "hyper_updated" has single values (as in the hyper_values) in all the json model config files that were shared with this problem, instead of a range of values. 

When I ran the demo.R example, I've noticed that the model json output "hyper_updated" had a range of values. eg:  "facebook_S_alphas": [0.5, 3]; 

I tried to manually change one of the "single values" in my specific failing case to a range of values in "hyper_updated" (eg. from "facebook_S_alphas": [2.9709] to "facebook_S_alphas": [0.5, 3]) and then I was able to run robyn_refresh.
